processor/internal/transformer/destination_transformer/destination_transformer_test.go

```diff
@@ -763,3 +768,125 @@ func TestLongRunningTransformation(t *testing.T) {
 		cancel()
 	})
 }
+
+func TestEmbeddedWarehouseTransformer(t *testing.T) {
+	pool, err := dockertest.NewPool("")
+	require.NoError(t, err)
+	transformerResource, err := transformertest.Setup(pool, t)
+	require.NoError(t, err)
+
+	conf := config.New()
+	conf.Set("DEST_TRANSFORM_URL", transformerResource.TransformerURL)
+	conf.Set("USER_TRANSFORM_URL", transformerResource.TransformerURL)
+	conf.Set("Processor.enableWarehouseTransformations", true)
+	conf.Set("Processor.verifyWarehouseTransformations", true)
+
+	ctx := context.Background()
+	eventsCount := 10000
+	trans := destination_transformer.New(conf, logger.NOP, stats.NOP)
+
+	clientEvents := lo.RepeatBy(eventsCount, func(index int) types.TransformerEvent {
+		return types.TransformerEvent{
+			Message: map[string]interface{}{
+				"src-key-1":         "test-value-1",
+				"messageId":         "messageId" + strconv.Itoa(index+1),
+				"event":             "event" + strconv.Itoa(index+1),
+				"originalTimestamp": "2021-09-01T00:00:00.000Z",
+				"receivedAt":        "2021-09-01T00:00:00.000Z",
+				"sentAt":            "2021-09-01T00:00:00.000Z",
+				"timestamp":         "2021-09-01T00:00:00.000Z",
+				"traits": map[string]any{
+					"email": float64(12),
```

Comment:
There is no email column in the output because of `skipTracksTable` being enabled ?